src/main/java/teammates/ui/pagedata/InstructorFeedbackResultsPageData.java

```diff
@@ -1202,6 +1194,57 @@ private void buildTableColumnHeaderForRecipientQuestionGiverView(List<ElementTag
         return responseRows;
     }
 
+    /**
+     * Adds comments associated with response and response giver/recipient index to response row.
+     *
+     * @param response response whose comments are to be added in responseRow
+     * @param question question associated with response
+     * @param responseRow response row of response
+     * @param responseGiverRecipientMap map in which key is both response recipient and response giver
+     *                                  and value is unique index.
+     */
+    private void addCommentsToResponseRow(FeedbackResponseAttributes response, FeedbackQuestionAttributes question,
```

Comment:
It's always a good practice to maintain an order of parameters.
In this case, should be something like `(question, response, responseRow, responseGiverRecipientMap`)

Also `responseGiverRecipientMap` -> `responseGiverRecipientIndexes`?